### ISSUE TITLE
关于训练出现ctcloss=0怎么解决

### ISSUE THREAD

**Issue by panlanlan** [kasyoukin/caffe_ocr_for_linux#41]
I0926 10:52:45.730175  5561 solver.cpp:237]     Train net output #0: ctcloss = 0 (* 1 = 0 loss)
I0926 10:52:45.730180  5561 sgd_solver.cpp:105] Iteration 3000, lr = 0.01
I0926 10:52:47.260527  5572 data_layer.cpp:78] Restarting data prefetching from start.
^CI0926 10:52:47.829211  5561 solver.cpp:447] Snapshotting to binary proto file densenet-res-blstm_train-val_iter_3104.caffemodel
